src/main/java/spoon/reflect/cu/position/DeclarationSourcePosition.java

```diff
@@ -15,8 +15,18 @@
  */
 public interface DeclarationSourcePosition extends CompoundSourcePosition {
 
+	/** returns the first char of the first modifier */
 	int getModifierSourceStart();
 
+	/** returns the last char of the last modifier */
 	int getModifierSourceEnd();
 
+	/** returns the end of the default value
+	 * int i = 0, j =1 => returns the comma
+	 */
+	int getDefaultValueEnd();
+
+	/** sets the position of the end of the default value declaration */
+	DeclarationSourcePosition setDefaultValueEnd(int endDefaultValueDeclaration);
```

Comment:
Not sure it is a good idea to make the position mutable. I think we paid attention in the past to keep them unmutable.

Reply:
I agree, immutability is such a good property. I'll change this.